lib/brakeman/app_tree.rb

```diff
@@ -215,6 +215,14 @@ def reject_global_excludes(paths)
       end
     end
 
+    def in_engine_paths?(path)
+      @engine_paths.any? { |p| path.absolute.include?(p) }
+    end
+
+    def in_add_libs_paths?(path)
```

Comment:
Note, I used the shorter convention seen in the CLI options interface: https://github.com/presidentbeef/brakeman/blob/0be81f6056b8e24f05da29546379fb079bdca573/lib/brakeman/options.rb#L190

I can rename this if desired to match the instance variable's name.